### ISSUE TITLE
Composite documention line needs punctuation and/or whitespace.

### ISSUE THREAD

**Issue by 0rir** [Raku/doc-website#176]
The introductory line following the header of composite pages is ill formed, e.g.:
```
rotor
Synthesised documentation from type/Supply type/List type/Any

```

**Comment by finanalyst** [comment]
The relevant line is in [Website/plugins/secondaries](https://github.com/Raku/doc-website/blob/327b86349db84eac04f2299eb61cd6df3f6fa616/Website/plugins/secondaries/gen-secondaries.raku#L87)
Would you please offer a PR to fix the issue?

**Comment by finanalyst** [comment]
addressed by #186 as well. So closing

**Comment by finanalyst** [comment]
@0rir Would you please look at PR #188?

A screen shot has been added to show changes that would address your concerns too.
